website/docs/components/form/masked-input/partials/code/component-api.md

```diff
@@ -47,7 +53,7 @@ The Masked Input component has two different variants with their own APIs:
   <C.Property @name="value" @type="string|number|date">
     Input control’s value.
   </C.Property>
-  <C.Property @name="isMasked" @type="boolean" @default="true">
+  <C.Property @name="isContentMasked" @type="boolean" @default="true">
```

Comment:
I _think_ we should document also the `@visibilityToggleAriaLabel` and `@visibilityToggleAriaMessageText` (passed down to the `::Base` element)